### PR TITLE
[10.0][MIG] port stock_ownership_availability_rules to 10.0

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**

### CODE REVIEW

**Review by lmignon — APPROVED** [review]
:+1:  LGTM (Code review)

File: stock_ownership_availability_rules/model/quant.py [hunk 1]
```diff
@@ -35,47 +35,29 @@ def create(self, vals):
...
                 location.company_id.partner_id.id or
-                Company.browse(
-                    Company._company_default_get('stock.quant')
-                ).partner_id.id
-            )
+                Company._company_default_get('stock.quant').partner_id.id)
 
         return super(Quant, self).create(vals)
 
     owner_id = fields.Many2one('res.partner', 'Owner',
```
Comment (carlosdauden) [inline stock_ownership_availability_rules/model/quant.py]: Maybe we could take advantage to put field declarations before CRUD methods:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models

**Review by carlosdauden — APPROVED** [review]
:+1: Minor comments

File: stock_ownership_availability_rules/model/quant.py (hunk 1 above)
Comment (Cedric-Pigeon) [reply stock_ownership_availability_rules/model/quant.py]: done

**Pull request merged**